### DISCUSSION TITLE
Comments: Angular AOT compilation with Webpack

### DISCUSSION THREAD

**Discussion by jiayihu** [jiayihu/blog#57] [Announcements]
This issue is reserved for comments to [Angular AOT compilation](http://blog.jiayihu.net/angular-aot-compilation-with-webpack/) with Webpack. Leave a comment below and it will be shown in the blog page.

**Comment by patelmurtuza** [comment]
Went through your article seems crystal clear. Can you please let me know if this article works with Angular5.2.3 and webpack3.8.1?

**Comment by jiayihu** [comment]
I've tested the examples using Angular 4 and webpack 2, so I think it should work since there haven't been big breaking changes in Angular compiler or webpack 3. You could check if the files mentioned in the article are still similar to those generated by latest `angular-cli`, whereas using the CLI `eject` command you can inspect the Webpack configuration.

Nevertheless I wrote this article one year ago, when `angular-cli` was still very unstable and even blocking people from deploying production code. Nowadays I believe it's much easier to just use the CLI, especially thinking about the new Ivy renderer and future compiler options.

**Comment by patelmurtuza** [comment]
Thanks, let me start.....
